app/src/main/java/com/example/moodtrackerproject/ui/WelcomeFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.moodtrackerproject.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import com.example.moodtrackerproject.R
+import com.example.moodtrackerproject.databinding.FragmentWelcomeScreenBinding
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class WelcomeFragment : Fragment() {
+
+    private var _binding: FragmentWelcomeScreenBinding? = null
+    private val mBinding get() = _binding!!
+    lateinit var navBar: BottomNavigationView
```

Comment:
Add `TODO`, this needs to be removed from the fragment and should be handled with LiveData